cli/eac.js

```diff
@@ -98,6 +98,16 @@ const getDefaultSchedulingValues = async () => {
   } 
 };
 
+const readUntilValid = (funcName) => {
+  while (true) {
```

Comment:
won't this lead to an eternal loop?

Reply:
It will look until it meets the return condition, which happens after they input a correct value.

Reply:
Might be a good idea to limit the loop to a couple of tries though.